src/plugins/intel_cpu/src/nodes/executors/acl/acl_mvn.hpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+// TODO: remove relative path
+#include "../mvn.hpp"
+#include "arm_compute/runtime/NEON/NEFunctions.h"
+
+namespace ov {
+namespace intel_cpu {
+
+class AclMVNExecutor : public MVNExecutor {
+public:
+    AclMVNExecutor();
+
+    bool init(const MVNAttrs& mvnAttrs,
+              const std::vector<MemoryDescCPtr>& srcDescs,
+              const std::vector<MemoryDescCPtr>& dstDescs,
+              const dnnl::primitive_attr &attr) override;
+    void exec(const std::vector<MemoryCPtr>& src, 
+              const std::vector<MemoryPtr>& dst, 
+              const void *post_ops_data_) override;
+
+    impl_desc_type getImplType() const override {
+        return implType;
+    }
+
+private:
+    impl_desc_type implType = impl_desc_type::acl;
+
+    arm_compute::Tensor srcTensor;
+    arm_compute::Tensor dstTensor;
+    std::unique_ptr<arm_compute::NEMeanStdDevNormalizationLayer> mvn = nullptr;
+};
+
+class AclMVNExecutorBuilder : public MVNExecutorBuilder {
+public:
+    bool isSupported(const MVNAttrs& mvnAttrs, 
+                     const std::vector<MemoryDescCPtr>& srcDescs, 
+                     const std::vector<MemoryDescCPtr>& dstDescs) const override {
+        if (mvnAttrs.epsMode_ == MVNEpsMode::INSIDE_SQRT) {
+            return false;
+        }
+        if (!mvnAttrs.normalizeVariance_) {
+            return false;
+        }
+        // ACL supports MVN with 2D inputs only
+        if (srcDescs[0]->getShape().getRank() != 2) {
```

Comment:
This limitation makes ACL executor almost useless as long as most of operations from real models have shape with rank >= 3.
From my undesranding ACL impls has 2D input shape [Y, X] and actual normalization is done over X axis, while Y serves as "batch". If my uderstanding correct we can translate any OV MVN shape into ACL representation using the following rules (example for 4D):
1. NCHW layout, mvnAttrs.initAcrossChannels_ == false; : Y = N*C, X = H*W
2. NCHW layout, mvnAttrs.initAcrossChannels_ == true; : Y = N, X = C*H*W
3. NHWC layout, mvnAttrs.initAcrossChannels_ == false; : unsupported
4. NHWC layout, mvnAttrs.initAcrossChannels_ == true; :  Y = N, X = H*W*C
I would suggest to try and check using functional single layer tests if it really works correclty

Reply:
@dmitry-gorokhov I updated PR - do I understand your point correctly? This code does not pass 4D tests because of accuracy issues.